cfg_mgmt/btp_service_binding.py

```diff
@@ -0,0 +1,159 @@
+import copy
+import json
+import logging
+import typing
+import requests
+
+import cfg_mgmt
+import cfg_mgmt.model as cmm
+import ci.log
+import ci.util
+import model
+import model.container_registry
+
+
+ci.log.configure_default_logging()
+logger = logging.getLogger(__name__)
+
+
+def _filter_service_bindings(result: dict, binding_name_prefix: str, instance_id: str) -> tuple[list[str], int]:
+    outdated_ids = []
+    next = 1
+    for item in result["items"]:
+        name = item["name"]
+        if name.startswith(binding_name_prefix) and item["service_instance_id"] == instance_id:
+            try:
+                n = int(name[len(binding_name_prefix):])
+                outdated_ids.append(item["id"])
+                if n >= next:
+                    next = n + 1
+            except ValueError:
+                logger.warn("ignored {}".format(name))
```

Comment:
nit: we consistently use single-quotes in this repository - at least, please do not mix within the same module